src/main/java/se/kth/sda/skeleton/comments/CommentRepository.java

```diff
@@ -2,10 +2,10 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
-
```

Comment:
I think we can revert the changes on this page as the spacing before is more standard.